pkg/siteconfig/clusterinstance.go

```diff
@@ -76,8 +129,121 @@ func PullClusterInstance(apiClient *clients.Settings, name, nsname string) (*Clu
 	return builder, nil
 }
 
+// WithExtraManifests includes manifests via configmap name.
+func (builder *CIBuilder) WithExtraManifests(extraManifestsName string) *CIBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return nil
```

Comment:
apologies for not noticing earlier, this should be `return builder`, which should fix the panic in the unit tests